build.gradle

```diff
@@ -190,6 +190,10 @@ repositories {
 
     maven { url 'https://maven.blamejared.com' }
 
+    // Temporary for Registrate

```

Comment:
This can be fully removed now; if its necessary in future I'll put it back with the proper URL :)